### DISCUSSION TITLE
ScriptReference/AnimationClip

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#139] [Notes]
# ScriptReference/AnimationClip

https://docs.unity3d.com/ScriptReference/AnimationClip.html

**Comment by vertxxyz** [comment]
If you're managing large animation clips (motion capture data for example), and notice that loading them is extremely slow, change your [asset serialization mode](https://docs.unity3d.com/Manual/class-EditorManager.html) to binary to improve load times significantly.
Note this will affect the whole project and will make source control more difficult.
